applications/nrf5340_audio/README.rst

```diff
@@ -1236,15 +1237,24 @@ You can test upgrading the firmware on both cores at the same time on a headset
 
       python buildprog.py -c both -b debug -d headset --pristine -m external
 
-#. Transfer the generated zip file to your Android or iOS device.
-   The file name should start with :file:`dev_headset_build_debug_dfu_application`.
+#. Transfer the generated file to your Android or iOS device.
+      a. For multi-image DFU, the file name follows the pattern ``dev_<Role>_build_<Application version>_dfu_application<Revision>.zip``,
+      For example :file:`dev_headset_build_debug_dfu_application_0.0.0+0.zip`.
+
+      b. For single-image DFU, the file for app core follows the pattern ``dev_<Role>_build_<Application version>_app_update_<Revision>.bin``
+      The file for net core follows the pattern ``dev_<Role>_build_<Application version>_net_core_app_update_<FW_VERSION>.bin``.
+      For example :file:`dev_headset_build_debug_app_update_0.0.0+0.bin`, :file:`dev_headset_build_debug_net_core_app_update_1.bin`.
+
+      The ``<Role>`` is ``headset`` or ``gateway``, ``<Application version>`` is ``debug`` or ``release``, ``<Revision>`` is :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION`,
```

Comment:
Maybe put this in a note?

``debug`` = ``build_debug``
``release`` = ``build_release``

Add "and" in stead of comma before ``<FW_VERSION>``